submissions/hannasyn/js-dom/index.js

```diff
@@ -0,0 +1,88 @@
+  },
+  {
+    id: 4,
+    navName: 'Fact number 4',
+    title: 'Calling yourself a feminist opens doors',
+    text: 'Women love feminism. By openly admitting you’re a feminist to your woman shows that you respect her just as much as you respect yourself. Whether you are an interesting or uninteresting man on the planet, if you stand behind feminism you will be awesome and more approachable! Feminism fact.'
+  },
+  {
+    id: 5,
+    navName: 'Fact number 5',
+    title: 'There is nothing that you need to do except think',
+    text: 'No one has the right to tell you what type of feminism you need to be. One of the first steps to feminism is changing the way you think about women in general. Try to picture having a wife with a full-time job and lastly, ask a woman if she wants to be touched by you instead of just doing it.'
+    }
+];
+
+const navList = document.querySelector('.navigation__list');
+const container = document.querySelector('.container');
+
+const paragraph = document.createElement('p');
+const title = document.createElement('h2');
+
+const asideBtn = document.querySelector('.aside-btn');
+const navigation = document.querySelector('.navigation');
+
+function generateText(){
+  title.classList.add('title-content');
+  paragraph.classList.add('content');
+  const startContent = 'Feminism is usually misunderstood and is viewed as a man-hating movement But it’s really quite the opposite! Feminism is about establishing equality between both men and women. Many women throughout history have undergone severe sexist laws thus the initiation of feminism movement was created to end discrimination. From eradicating female genital disfigurement to setting “girl power” in the 90’s. Relentless, stimulating, lively, serious, provocative, and above all empowering.';
+  paragraph.textContent = startContent;
+  container.append(paragraph);
+};
+
+function createNavItem() {
+  facts.forEach(function(fact) {
+    const navItem = document.createElement('li');
+    navItem.classList.add('navigation__list-item');
+    navItem.innerHTML = `<a data-id="${fact.id}" class="navigation__list-link" href="#">${fact.navName}</a>`;
+    navList.append(navItem);
+  })
+};
+
+function displayFact(navLink) {
+  facts.find(function(fact) {
+    if (navLink.dataset.id == fact.id) {
+        title.innerText = fact.title;
+        paragraph.innerText = fact.text;
+        container.append(title);
+        container.append(paragraph);
+      }
+  }) 
+};
+
+navList.addEventListener('click', function(e) {
+  let target = event.target;
+  e.preventDefault();
+  const navItemActive = document.querySelector('.navigation__list-item.active');
+  if (navItemActive) {
+    navItemActive.classList.remove('active');
+  }
+  target.parentNode.classList.add('active');
+  displayFact(target);
+});
+
+generateText();
```

Comment:
You can wrap calling these functions into some function for initing your app and call this function after loading your DOM tree. 
More info [here](https://developer.mozilla.org/uk/docs/Web/Events/DOMContentLoaded)

Reply:
fixed